native/src/components/Consent.tsx

```diff
@@ -0,0 +1,59 @@
+import { useFocusEffect } from '@react-navigation/native'
+import React, { ReactElement, useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { View, StyleSheet } from 'react-native'
+import styled from 'styled-components/native'
+
+import { capitalizeFirstLetter } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import appSettings, { ExternalSourcePermission } from '../utils/AppSettings'
+import { updateSourcePermissions } from '../utils/helpers'
+import { reportError } from '../utils/sentry'
+import Caption from './Caption'
+import ConsentSection from './ConsentSection'
+import Layout from './Layout'
+
+export const ItemSeparator = styled.View`
+  background-color: ${props => props.theme.colors.textDecorationColor};
+  height: ${StyleSheet.hairlineWidth}px;
+`
```

Comment:
Optional: We already have some Divider/Separator components in use (e.g. HorizontalLine, Divider in two files). Maybe we should merge them at some point as well.